src/Filesystem/Filesystem.php

```diff
@@ -120,49 +120,57 @@ public function removeFile(Media $media, string $path)
         $this->filesystem->disk($media->disk)->delete($path);
     }
 
-    public function syncFileNames(Media $media)
+    public function syncFiles(Media $media)
```

Comment:
Renaming this function is a breaking change. Please add `syncFileNames` back. It should call this new `syncFiles` function.